notebooks/2_read_write.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>An answer needs to be added here?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>First idea was to explain the [TAB] idea to explore the read_* namespace but as we can't really take any assumptions on the IDE of the user, I excluded this.</p>